wordpress-mobile/gutenberg-mobile
# Fix: prevent ripple effect on slider cell in BottomSheet and disable it completely on iOS

Fixes #2022

Please also refer to:
[Related gutenberg PR](https://github.com/WordPress/gutenberg/pull/20922)
[Related gutenberg-mobile issue](https://github.com/wordpress-mobile/gutenberg-mobile/issues/2022)

To test:
1. Add `Spacer` block
2. Open `Spacer` settings
3. Press on Slider row title
4. See if there is no ripple effect 

- check if a11y works after proposed changes

PR submission checklist:
